vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "Sourcing tool_installs_mariner.sh"
+
+installAscBaseline() {
+   echo "Mariner TODO: installAscBaseline"
+}
+
+installBcc() {
+    echo "Installing BCC tools..."
+    apt_get_update || exit $ERR_APT_UPDATE_TIMEOUT
+    apt_get_install 120 5 25 bcc-tools kernel-headers-$(uname -r) || exit $ERR_BCC_INSTALL_TIMEOUT
+}
+
+configGPUDrivers() {
+    echo "Not installing GPU drivers on Mariner"
+}
+
+disableSystemdTimesyncdAndEnableNTP() {
```

Comment:
this method is only for ubuntu 1804 , we can remove it and add one that enables chrony (and only call it for Mariner)

Reply:
systemd-timesyncd is disabled by default in favor of chrony, so I'll just remove this function from Mariner altogether.